packages/neon-core/__integration__/rpc/clients/NeoServerRpcClient.ts

```diff
@@ -226,6 +226,52 @@ describe("NeoServerRpcClient", () => {
     expect(result).toBe("AOH1BQ==");
   });
 
+  describe.only("findStorage", () => {
+    test("find NEOs totalSupply storage", async () => {
+      const findTotalSupplyPrefixResult = await client.findStorage(
+        contractHash,
+        "0b",
+        0
+      );
+
+      const findTotalSupplyPrefixWithDefaultArgResult =
+        await client.findStorage(contractHash, "0b");
+
+      // This find will only get the totalSupply of NEO. Should be safe and static for usage.
+      expect(findTotalSupplyPrefixResult.truncated).toBe(false);
+      expect(findTotalSupplyPrefixResult.next).toBe(1);
+      expect(findTotalSupplyPrefixResult.results.length).toBe(1);
+      expect(findTotalSupplyPrefixResult.results[0].key).toBe("Cw==");
+      expect(findTotalSupplyPrefixResult.results[1].key).toBe("AOH1BQ==");
```

Comment:
`TypeError: Cannot read properties of undefined (reading 'key')`

Now that I look closer, results.length is 1 so you cannot retrieve the [1] of it.